pkgs/development/libraries/physics/fastnlo_toolkit/default.nix

```diff
@@ -53,6 +38,10 @@ stdenv.mkDerivation rec {
   preConfigure = ''
     substituteInPlace ./fastnlotoolkit/Makefile.in \
       --replace "-fext-numeric-literals" ""
+
+    # disable test that fails due to strict floating-point number comparison
+    substituteInPlace ./check/Makefile.in \
+      --replace "fnlo-tk-stattest.pl" ""
```

Comment:
Might be more robust than patching Makefiles:

```suggestion
    echo "#!/usr/bin/env perl" > check/fnlo-tk-stattest.pl.in
    chmod +x check/fnlo-tk-stattest.pl.in
```

Reply:
Sure, that will report test as if it ran, though